examples/webgl_loader_vrml.html

```diff
@@ -1,129 +1,132 @@
-				controls.zoomSpeed = 5;
-
-				scene = new THREE.Scene();
-				scene.add( camera );
-
-				// light
-
-				var dirLight = new THREE.DirectionalLight( 0xffffff );
-				dirLight.position.set( 200, 200, 1000 ).normalize();
-
-				camera.add( dirLight );
-				camera.add( dirLight.target );
-
-				var loader = new THREE.VRMLLoader();
-				loader.load( 'models/vrml/house.wrl', function ( object ) {
-
-					scene.add( object );
-
-				} );
-
-				// renderer
-
-				renderer = new THREE.WebGLRenderer( { antialias: false } );
-				renderer.setPixelRatio( window.devicePixelRatio );
-				renderer.setSize( window.innerWidth, window.innerHeight );
-
-				container = document.createElement( 'div' );
-				document.body.appendChild( container );
-				container.appendChild( renderer.domElement );
-
-				stats = new Stats();
-				container.appendChild( stats.dom );
-
-				//
-
-				window.addEventListener( 'resize', onWindowResize, false );
-
-			}
-
-			function onWindowResize() {
-
-				camera.aspect = window.innerWidth / window.innerHeight;
-				camera.updateProjectionMatrix();
-
-				renderer.setSize( window.innerWidth, window.innerHeight );
-
-				controls.handleResize();
-
-			}
-
-			function animate() {
-
-				requestAnimationFrame( animate );
-
-				controls.update();
-				renderer.render( scene, camera );
-
-				stats.update();
-
-			}
-
-		</script>
-
-	</body>
+<head>
+    <title>VRML loading with ThreeJs and VrmlParser</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+    <style>
+        body {
+            font-family: Monospace;
+            background-color: #000;
+            color: #fff;
+            margin: 0px;
+            overflow: hidden;
+        }
+
+        #info {
+            color: #fff;
+            position: absolute;
+            top: 10px;
+            width: 100%;
+            text-align: center;
+            z-index: 100;
+            display: block;
+        }
+
+        #info a, .button {
+            color: #f00;
+            font-weight: bold;
+            text-decoration: underline;
+            cursor: pointer
+        }
+
+        #viewpoints {
+            background-color: gray;
+            z-index: 101;
+            position: absolute;
+            width: 140px;
+            top: 10px;
+            left: 10px;
+            padding: 2px;
+            border: 1px solid white;
+        }
+
+        #viewpoints div {
+            width: inherit;
+        }
+    </style>
+
+</head>
+
+<body>
+<div id="info">
+    <a href="http://threejs.org" target="_blank">three.js / VrmlParser</a> -
+    vrml format loader test using VrmlParser -
+</div>
+<div id="viewpoints">
+    <h4>Viewpoints from the VRML file</h4>
+</div>
+
+<script src="../build/three.min.js"></script>
+<script src="js/libs/vrml.min.js"></script>
+
+<script src="js/loaders/VRMLLoader.js"></script>
+<script src="js/renderers/Projector.js"></script>
```

Comment:
@makc Okay, now I'm getting it and I totally agree, but this is just an example. You can use all of the components separately. This example is just trying to demonstrate that you can load a VRML file using Three. If for example you would want to just get back the tree, you can use the vrmlParser from the VrmlParser project. And it is up to you if you want to use the ThreeJs renderer that it offers and if you want animation support or not. The whole VrmlParser by the way is somewhat experimental and needs improvement, but it is better than what we had as a Vrml example in three so far. As it improves, I will also update the example here.

It also won't do stuff with the camera, unless your VRML file specifies ViewPoint. In that case, if you click one of the ViewPoints, the camera will go there, if you specified a container id for the ViewPoints list.